plugin/vim-tmux-runner.vim

```diff
@@ -246,7 +246,12 @@ function! s:AltPane()
   endif
 endfunction
 
-function! s:PromptForRunnerToAttach()
+function! s:PromptForRunnerToAttach(...)
+  let s:pane_index = ''
+  if exists("a:1") && a:1 != ""
+      let s:pane_index = a:1
+  endif
+  if s:pane_index == ''
```

Comment:
I'm not quite following the logic here. Should this conditional be merged with the next line?

Reply:
Hi @christoomey,
The condition placement is correct but the indentation is wrong [my bad]. I will correct it and raise a new PR.

To explain further - [ if s:pane_index == ''  ] this check is for identifying if its interactive or non-interactive method of attaching the pane. The condition in the next line is to check if Runner must be attached to the alternate pane if only 2 panes exist or display the pane numbers if more than 2 panes exist in the window.

I wanted to keep the existing behaviour untouched, combining both the conditions would probably lead to complicated logical check's.

Reply:
Hi @christoomey,
The condition placement is correct but the indentation is wrong. Anyways I have simplyfied it further. I will raise a new PR for this and the below review.

Reply:
Hi @vinodkri, thanks for the reply. Just to be super clear, you can push any changes or updates to your existing branch, no need to open a new PR.

Reply:
Yep! will do.